<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentWork</class>
 <widget class="QMainWindow" name="StudentWork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Студенческий состав</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item row="14" column="0" colspan="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Имя</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QComboBox" name="gru"/>
      </item>
      <item row="19" column="0" colspan="3">
       <widget class="QPushButton" name="addStuBut">
        <property name="text">
         <string>Добавить студента</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Выберите новую группу</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="16" column="0" colspan="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Отчество</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="3">
       <widget class="QLineEdit" name="stuSurname"/>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Менеджер работы с студентами</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QComboBox" name="stu"/>
      </item>
      <item row="12" column="0" colspan="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Фамилия</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QComboBox" name="moveNewGru"/>
      </item>
      <item row="18" column="0" colspan="3">
       <widget class="QComboBox" name="pickGru"/>
      </item>
      <item row="6" column="1" rowspan="3">
       <widget class="Line" name="line_9">
        <property name="midLineWidth">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="20" column="0" colspan="3">
       <widget class="QLabel" name="statusStu">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="moveStuBut">
        <property name="text">
         <string>Переместить между группами</string>
        </property>
       </widget>
      </item>
      <item row="17" column="0" colspan="3">
       <widget class="QLineEdit" name="stuPar"/>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Список студентов</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="15" column="0" colspan="3">
       <widget class="QLineEdit" name="stuName"/>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Список групп</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="3">
       <widget class="Line" name="line_8">
        <property name="midLineWidth">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="addActive">
        <property name="text">
         <string>Отметить активного студента</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="removeStuBut">
        <property name="text">
         <string>Отчислить студента</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="cancelActive">
        <property name="text">
         <string>Забрать активность студента</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="back">
      <property name="text">
       <string>Вернуться</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>466</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
